yapi.py

```diff
@@ -38,44 +39,58 @@
 no_answer = ("N", "No", "n", "no")
 right_answer = yes_answer + no_answer
 
-continue_to_ask = True
-
-while continue_to_ask:
+if len(sys.argv) == 1:
+    continue_to_ask = True
+    while continue_to_ask:
+        print("-" * 79)
+        print("You can choose to install this packages:")
+        for package_counter in packages:
+            print("{:>2}) {} - {}".format(
+                package_counter,
+                packages[package_counter][0],
+                packages[package_counter][1]))
+        choose = -1
+        while choose not in range(0, package_counter + 1):
+            choose = int(input("What package do you want to install? "))
+        package_to_install = choose
+        while choose not in right_answer:
+            choose = input(
+                "Do you want to install {}? ".format(
+                    packages[package_to_install][0]))
+        if choose in yes_answer:
+            print("Let's start the installation, these take a moment...")
+            with open(packages[package_to_install][2], "r") as file_script:
+                bashCommand = ""
+                for line in file_script.readlines():
+                    if line[0] != "#":
+                        bashCommand += line
+                bashCommand = bashCommand.replace("\n", " ; ")
+                subprocess.call(bashCommand, stderr=subprocess.STDOUT, shell=True)
+        else:
+            print("Ok, no problem...")
+        choose = ""
+        while choose not in right_answer:
+            choose = input("Do you want to install something else? ")
+        if choose in yes_answer:
+            continue
+        else:
+            print("Ok, bye bye!")
+            break
     print("-" * 79)
-    print("You can choose to install this packages:")
-    for package_counter in packages:
-        print("{:>2}) {} - {}".format(
-            package_counter,
-            packages[package_counter][0],
-            packages[package_counter][1]))
-    choose = -1
-    while choose not in range(0, package_counter + 1):
-        choose = int(input("What package do you want to install? "))
-    package_to_install = choose
-    while choose not in right_answer:
-        choose = input(
-            "Do you want to install {}? ".format(
-                packages[package_to_install][0]))
-    if choose in yes_answer:
+
+elif len(sys.argv) == 2:
+    print("You must provide another argument for the package or no arguments for the question installer")
+    print("python yapi.py install package for example")
+
+elif len(sys.argv) == 3:
+    if sys.argv[1] == "install":
+        package_to_install = sys.argv[2]
         print("Let's start the installation, these take a moment...")
-        with open(packages[package_to_install][2], "r") as file_script:
+        with open("scripts/" + sys.argv[2] + ".sh", "r") as file_script:
```

Comment:
I know that's the same, but "scripts/" is store in where_is_scripts variable. This directory may change in the future, so use a variable is less problematic because there will be less to change.

Reply:
I will change that